src/bin/cli/harmonic.js

```diff
@@ -37,31 +37,37 @@ program
     });
 
 program
-    .command('new_post ["title"] [path]')
+    .command('new_post <title> [path]')
+    .option('--no-open', 'Don\'t open the markdown file(s) in editor')
     .description('Create a new post')
     // [BUG] https://github.com/jshint/jshint/issues/1849 - can't use arrow function
-    .action(function(title, path = '.') {
-        newFile(path, 'post', title);
+    // [BUG] https://github.com/jshint/jshint/issues/1779#issuecomment-68985429
+    .action(function(title, path = '.', { open: autoOpen }) { // jshint ignore:line
```

Comment:
When we define `.option('--no-open', ...)`, Commander.js creates a boolean option called "open" which defaults to `true` and is only false when the `--no-open` flag is present.

Relevant source:
- https://github.com/tj/commander.js/blob/29b4ea91a18679546980ce1a1e07957d1cbf9fb5/index.js#L316-L317
- https://github.com/tj/commander.js/blob/29b4ea91a18679546980ce1a1e07957d1cbf9fb5/index.js#L356-L362